README.md

```diff
@@ -59,6 +59,10 @@ prior to issuing the command above. ```<properties>``` refers
 to one or more property (i.e. *.mop) files, or a directory containing
 such property files.
 
+Note that when building an agent, users need to have their properties
```

Comment:
Hi,

I&#39;d like to add you to my professional network on LinkedIn.

- Qingzhou

Accept: https://www.linkedin.com/blink?simpleRedirect=0Ud3gSdjoTe30ScP8MejsRdPARfkh9rCZFt65QqnpKqioMrj1JemJElPRVpkJApn9xq7cCt7dBtmtvpnhFtCVFfmJB9C5QoORBt6BSrCAJt7dBtmsJr6RBfmtKqmJzon9Q9ClQqnpKimtBkClOs3Rx9z8NbmlHp39IqjpFbn9Lq7pPu3RBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&msgID=I5975790254982987777_500&markAsRead=


You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/blink?simpleRedirect=cz4JpmJAcCNFdCAJsCZEtDdUfmhFpipHp7lyh3hBuANVpkNjj3BniiRMen8MdnBxj4kSbkNkpj1Uq6VMdS4VuCBWcSJLt4xlpP5KomtMr5BDtkVUmk5FgnxBnPsQhQt3jBtUmlhht6Rygz1Sin9Ld7h1cB5emkVEdQdiikIJsklFp7dBj4tkrnd9q7Bde4R4mAtIcQcVjkZRiz5kqkRbll9nmjtIhzlRlk51gmtmiSZFtSJMsDFihl51fmhFrSMCt7dBtmtvpnhFtCVFfmJB9B4ScSh5hSxmhD9JhB51fmVBqSZkp6BJ9CVRr3RQ9z8NbmlHp39IqjpFbn9Lq7pPu3RBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&amp;msgID=I5975790254982987777_500&amp;markAsRead= Learn why we included this at the following link: https://www.linkedin.com/blink?simpleRedirect=e3wTd3RAimlIoSBQsC4Ct7dBtmtvpnhFtCVFfmJB9CNOlmlzqnpOpldOpmRLt7dRoPRx9z8NbmlHp39IqjpFbn9Lq7pPu3RBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&msgID=I5975790254982987777_500&markAsRead=
&copy; 2014, LinkedIn Corporation. 2029 Stierlin Ct. Mountain View, CA 94043, USA